scripts/build.py

```diff
@@ -86,6 +88,13 @@ def parse_args():
         '--code-style', dest='code_style',
         action='store_true', default=False,
         help='Apply C++ code style before compiling')
+    arg_parser.add_argument(
+        '--skip-tests', dest='skip_tests',
+        action='store_true', default=False,
```

Comment:
I agree that it's not needed, but it makes it clear what's going on, which is good for script maintenance.

Reply:
Unless you feel strongly about this, I would prefer to leave it as is.

Reply:
Just for the record... what you find "makes it clear what's going on", I find "misleading because it makes me feel something is deliberately different from the default" and "misleading because it's an unusual usage of parameters".  I don't memorize all the arguments to every library function, so when I see unexpected ones being used, I have to go look them up.  So for me this is more effort spent every time someone reads the code, just to discover that the parameters are all equal to defaults.

But admittedly this doesn't affect code function and, all things considered, this is relatively minor.  Your preference is acceptable.

As an alternative suggestion (which you may also ignore :-), a small comment above all the `add_argument()` calls stating your intention, e.g.:
```
# Many of the arguments here have default values; there are provided to
# make the action explicitly clear.
```
would make it clear that this was deliberately done and why, and would lessen the confusion of code readers like me.  It makes it clear you know these are defaults (so I don't have to point that out in case it was an oversight) and doesn't make me wonder why the defaults are being provided.  (I'd still have to look up the library function to make sure the arguments are being used correctly.)

But if you don't want to provide such a comment, that's acceptable too.